### ISSUE TITLE
The header name random.h is confusing.

### ISSUE THREAD

**Issue by p1v0t** [RayTracing/InOneWeekend#77]
As C standard library provide random.h, defining a header with same name is problematic and should be avoided.

**Comment by ronaldfw** [comment]
There is no C standard library header called "random.h" (see https://en.m.wikipedia.org/wiki/C_standard_library). There is a C++11 header called "random" (no .h). Given that, do you still find the name problematic?

I like "random.h" because it is short and succinct. 

What do others think?

**Comment by hollasch** [comment]
I agree with Ronald. In particular, the inclusion is via double quotes instead of angle brackets, which typically means local include path rather than a system path.

This _would_ be an issue if we were shipping this code, and if it was intended to be used as a library. Instead, it's intended as an example implementation, and the reader is strongly encourage to write the code themselves.

**Comment by vchizhov** [comment]
The main issue with random.h is that it uses rand().

**Comment by ronaldfw** [comment]
@vchizhov Would you mind elaborating on this further?

There's some discussion on https://github.com/RayTracing/InOneWeekend/issues/68 regarding rand() vs using C++ `<random>` and speed (I found rand() to be faster). I also compared result images and did not see any notable differences.

**Comment by vchizhov** [comment]
@ronaldfw
There's a wealth of articles describing why rand() is bad for simulations. Long story short: it's usually an lcg prng, fails multiple statistical tests, and the % only adds to the problems.
Here's some articles:
https://scicomp.stackexchange.com/questions/30479/how-good-are-current-implementations-of-rand-in-c
http://www.reedbeta.com/blog/quick-and-easy-gpu-random-numbers-in-d3d11/
http://www.pcg-random.org/
